components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -270,7 +270,8 @@ def _unwrapunits(self, obj, units=None):
             return obj
         return obj.getValue()
 
-    def _getQueryString(self, opts=None):
+    @classmethod
+    def _getQueryString(klass, opts=None):
```

Comment:
I don't understand the usage of `klass`, http://legacy.python.org/dev/peps/pep-0008/#function-and-method-arguments